roles/pulp_common/tasks/install_pip.yml

```diff
@@ -260,6 +260,18 @@
       tags:
         - molecule-idempotence-notest
 
+    - name: Get pulpcore version
+      shell: "{{ pulp_install_dir }}/bin/pip3 show pulpcore | grep Version | cut -d' ' -f2"
+      args:
+        executable: /bin/bash
+      register: pulpcore_version
+      check_mode: false
+      changed_when: false
+
+    - name: Set __pulp_version to the pulpcore that was installed
+      set_fact:
+        __pulp_version: "{{ pulpcore_version.stdout }}"
+
```

Comment:
1. So __pulp_version is the exact version of pulp that got installed, unlike __pulpcore_version, correct? (Dennis's PR.)

2. We obtain the pulpcore version in the task after this one. See the task `Upgrade dynaconf to 3.1.1rc2 pre-release if it is older than 3.1.1` in this file for an example of how to access that variable. The `.dynaconf[0]` becomes `.pulpcore[0]`